sdk/iot/README.md

```diff
@@ -1,14 +1,38 @@
-# Azure IoT Clients for Embedded C
+# Embedded C for Azure IoT Clients
 
-Official Embedded C clients for Azure IoT Hub and Device Provisioning Service.
+Official Embedded C clients for Azure IoT Hub services.
 
 ## Getting started
 
-TBD
+- **Docs**: If you're new to the Azure IoT Embedded C SDK take a look at the [Machine State diagram](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/iot/doc/mqtt_state_machine.md#device-provisioning-service) that explains the high-level architecture, SDK components, and a clear view of SDK x Application responsibilities. 

```

Comment:
"State Machine"

Reply:
should the last line read

```
...and a clear view of application responsibilities
```

?

I feel like the previous item (SDK components) explains what the SDK does and then we state what are the responsibilities of the app code.

Reply:
The link is pointing at Device Provisioning - we could point at the top of the document instead.

Reply:
Also, it may be simpler to just remove that document entirely and have the diagram and most of the document in the top-level readme.md